src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    int areaCalculator();
```

Comment:
Method should represent action it does and have the proper name. Looks like this method returns some calculator of area. Please, give some proper name like "calculateArea" or "getArea"